interface/forms/procedure_order/new.php

```diff
@@ -212,6 +213,10 @@ function getListOptions($list_id , $fieldnames=array('option_id', 'title', 'seq'
 <html>
 <head>
 <?php html_header_show(); ?>
+<script src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-min-1-9-1/index.js"></script>
+<script src="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/js/bootstrap.min.js"></script>
```

Comment:
place this line below the line (249) below where the other js libraries are brought in